### PR TITLE
Improves OOP exercise instructions

### PR DESCRIPTION
**Pull request by Genyus**
- Adds test exclusion instructions for shapes and accounts exercises
- Adds instructions to run application code
- Improves heading labels

### CODE REVIEW

**Review by tanyapowell — APPROVED** [review]

File: exercises/object-oriented-programming/README.md [hunk 1]
```diff
@@ -86,6 +89,17 @@ mvnw --projects cars clean test -Dtest=ShowroomTest
 
 Your implementation is correct when all tests pass.
 
+#### :information_source: Notes
+If you want to experiment with the provided application in the App.java file, you can run a command in this format `./mvnw -q --projects [project name] clean compile exec:java -Dexec.mainClass=[package].[class]` from the terminal:
```
Comment (tanyapowell) [inline exercises/object-oriented-programming/README.md:93]: Is this command different on Windows?

File: exercises/algorithms/search/src/main/java/com/cbfacademy/search/Search.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.cbfacademy.search;
+
+public interface Search {
+    /**
+     * Searches an array.
+     *
+     * @param sortedArray The array to be searched (the array should already be s
+     *                    rted)
```
Comment (tanyapowell) [inline exercises/algorithms/search/src/main/java/com/cbfacademy/search/Search.java:8]: This formatting is a bit weird

**Review by tanyapowell — CHANGES_REQUESTED** [review]

File: exercises/object-oriented-programming/README.md (hunk 1 above)
Comment (Genyus) [reply exercises/object-oriented-programming/README.md:93]: Not to my knowledge

**Review by tanyapowell — APPROVED** [review]

File: exercises/object-oriented-programming/README.md [hunk 3]
```diff
@@ -176,12 +160,6 @@ In your terminal, run the following command:
 ./mvnw --projects accounts clean test -Dtest=CurrentAccountTest,SavingsAccountTest
 ```
 
-If you are on Windows, run this command instead:
-
-```bat
```
[the remark this answers is not in the document]
Comment (Genyus) [reply exercises/object-oriented-programming/README.md:153]: Eventually yes, although I'd say it's low priority. We've been pushing Windows students to use the Git terminal, which uses the same Linux-style commands, but it's not formalised, so we should clearly define our policy in this area first

**Pull request merged**